securedrop/journalist_templates/edit_account.html

```diff
@@ -72,7 +72,7 @@ <h2>{{ gettext('Reset Password') }}</h2>
     <br>
     <span id="password" class="password">{{ password }}</span>
   </p>
-  <button class="sd-button btn" type="submit" id="reset-password"><i class="fas fa-sync"></i> {{ gettext('Reset Password') }}</button>
+  <button type="submit" id="reset-password"><i class="fas fa-sync"></i> {% filter upper %}{{ gettext('Reset Password') }}{% endfilter %}</button>
```

Comment:
Will this work with all locales/languages? We should explicitly test this as part of QA